api/src/main/java/org/openmrs/module/sync2/api/utils/SyncUtils.java

```diff
@@ -165,7 +172,28 @@ public static Class getClass(String client, String category) {
         }
     }
 
+    public static boolean compareLocalAndPulled(String clientName, String category, Object from, Object dest) {
+        boolean result = false;
 
+        if (null != dest && !(from instanceof String)) {
+            switch (clientName) {
+                case REST_CLIENT_KEY:
+                    RestResource restLocal = RestResourceCreationUtil.createRestResourceFromOpenMRSData((OpenmrsObject) dest);
+                    RestResource restPulled = RestResourceCreationUtil.createRestResourceFromOpenMRSData((OpenmrsObject) from);
```

Comment:
There is something wrong with naming. I don't think that it is needed to know which object is local and which a parent object. So they can be named easily, for example, object1 and object2.